Tests/SwiftDocCTests/Semantics/SnippetTests.swift

```diff
@@ -58,10 +58,19 @@ class SnippetTests: XCTestCase {
         XCTAssertNotNil(snippet)
         XCTAssertTrue(problems.isEmpty)
     }
-
-    func testNoTopLevelPageForSnippetModule() throws {
-        let (_, context) = try testBundleAndContext(named: "TestBundle")
-        XCTAssertEqual(3, context.rootModules.count)
-        XCTAssertFalse(context.rootModules.contains { $0.path == "/documentation/Test" })
+    
+    func testSliceResolves() throws {
```

Comment:
Are we missing a test that verifies a diagnostic is emitted if you're referring to a snippet that doesn't exist?

Reply:
Added a test.